Library/Homebrew/cmd/style.rb

```diff
@@ -1,16 +1,12 @@
-#:  * `style` [`--fix`] [`--display-cop-names`] [<formulae>|<files>]:
+#:  * `style` [`--fix`] [`--display-cop-names`] [<formulae>|<files>|<taps>]:
```

Comment:
It's not a major departure from the status quo, but will users be confused that they can specify multiple items of the same kind (e.g. two taps) but are not allowed to mix and match (e.g. a file name and a formula name)? Can we address this somehow to make this more user-friendly?